modules/drivers/mongo/src/metabase/driver/mongo/util.clj

```diff
@@ -225,13 +229,14 @@
     (ssh/with-ssh-tunnel [details-with-tunnel details]
       (let [connection-info (details->mongo-connection-info (normalize-details details-with-tunnel))
             [mongo-client db] (connect connection-info)]
-       (log/debug (u/format-color 'cyan (trs "Opened new MongoDB connection.")))
-       (try
-         (binding [*mongo-connection* db]
-           (f *mongo-connection*))
-         (finally
-           (mg/disconnect mongo-client)
-           (log/debug (u/format-color 'cyan (trs "Closed MongoDB connection.")))))))))
+        (log/debug (u/format-color 'cyan (trs "Opened new MongoDB connection.")))
+        (try
+          (binding [*mongo-connection* db
+                    *mongo-client*     mongo-client]
```

Comment:
I added this to dynamic scope to make it available up [here](https://github.com/metabase/metabase/pull/35964/files#diff-e7783632d76b787db2581115d762107f65336b44af25298042df2da4ac1a7731R60). Call me lazy, but I didn't want to break the macro interface by making this an argument. I think it's worth it.